source/classes/warlock.rst

```diff
@@ -45,51 +45,75 @@ Equipment
 -  Leather armor, any simple weapon, and two daggers
 
 The Warlock
-^^^^^^^^^^^
+~~~~~~~~~~~
+
++-------+-------------+-----------------------------------------+----------+---------+-----------------------+
+|       |             |                                         |          |         | Spell Slots per       |
```

Comment:
Add the `---` on either side of `spell slots per level` like in the other caster classes